portfolio/src/main/webapp/style.css

```diff
@@ -112,10 +117,12 @@ a {
 
 .card:hover>.card-subheading {
   filter: invert(0);
+  display:initial;
```

Comment:
add a space after `:`

Reply:
Rectified in commit here and in other places with 4c79bad